src/Symfony/Component/Form/Util/OrderedHashMap.php

```diff
@@ -96,12 +95,13 @@ class OrderedHashMap implements \ArrayAccess, \IteratorAggregate, \Countable
     /**
      * Creates a new map.
      *
-     * @param array<TKey, TValue> $elements The elements to insert initially
+     * @param array<string, TValue> $elements The elements to insert initially
```

Comment:
should be just `array<TValue>` here as we accept any kind of array-key here, and we don't need to make it explicit

Reply:
Which we generally write as `TValue[]`

Reply:
Changed